src/Microsoft.AspNetCore.Routing/RouteEndpoint.cs

```diff
@@ -7,56 +7,47 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Routing.Patterns;
 
-namespace Microsoft.AspNetCore.Routing.Matching
+namespace Microsoft.AspNetCore.Routing
 {
     /// <summary>
     /// Represents an <see cref="Endpoint"/> that can be used in URL matching or URL generation.
     /// </summary>
-    public sealed class MatcherEndpoint : Endpoint
+    public sealed class RouteEndpoint : Endpoint
     {
-        internal static readonly Func<RequestDelegate, RequestDelegate> EmptyInvoker = (next) =>
-        {
-            return (context) => Task.CompletedTask;
-        };
+        internal static readonly RequestDelegate EmptyRequestDelegate = (context) => Task.CompletedTask;
```

Comment:
This should be in the base

Reply:
Then it moves to the HttpAbstractions repo, and we replace it with something else.

Reply:
It's just used by tests. I'm going to shift it to a helper class in a test project.